homeassistant/components/sector_alarm/sensor.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from datetime import timedelta
+from homeassistant.components.sector_alarm import DOMAIN as SECTOR_DOMAIN
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+from homeassistant.const import TEMP_CELSIUS
+
+UPDATE_INTERVAL = timedelta(minutes=5)
+
+DEPENDENCIES = ['sector_alarm']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Initialition of the platform."""
+    sector_connection = hass.data.get(SECTOR_DOMAIN)
```

Comment:
Setting up platforms sensor and alarm panel differs, sensor is lacking some verifications. Standardise these